common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -373,11 +378,8 @@ export interface ISignalMessage extends ISignalMessageBase {
  * Interface for signals sent by clients to the server when submit_signals_v2 is enabled.
  * @internal
  */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
You can use `type foo = bar` here. (No lint disables if you can help it.)
The comment is also funny now - this type isn't special in that regard.